packages/nonMenuDropdown/NonMenuDropdown.tsx

```diff
@@ -0,0 +1,148 @@
+import * as React from "react";
+import { useId } from "react-id-generator";
+import FocusLock from "react-focus-lock";
+import { Dropdownable } from "../dropdownable";
+import { Popover } from "../popover";
+import { SpacingBox } from "../styleUtils/modifiers";
+import { DropdownableProps } from "../dropdownable/components/Dropdownable";
+import { ResetButton } from "../button";
+import { display } from "../shared/styles/styleUtils";
+
+interface NonMenuDropdownProps
+  extends Omit<DropdownableProps, "open" | "dropdown" | "onClose"> {
+  /**
+   * A unique identifier to help screenreaders or UI testing tools associate the dropdown content with it's trigger button. If an ID prop is not provided, an arbitrary ID will be generated.
+   */
+  id?: string;
+  /**
+   * Whether the dropdown starts open
+   */
+  initialIsOpen?: boolean;
+  /** The maximum height of the dropdown overlay, represented by number of pixels. */
+  maxHeight?: number;
```

Comment:
might there ever be the case that a `vh` could come in handy? in that case a string might also be a valid choice.

Reply:
There probably is a case, but I can't think of one right now. If we find a case for accepting strings with CSS units, we should do a follow-up PR that adds that functionality to all Popover-like components